activeadmin.gemspec

```diff
@@ -18,7 +18,16 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = %w[CHANGELOG.md CODE_OF_CONDUCT.md CONTRIBUTING.md README.md]
 
-  s.metadata = { "rubygems_mfa_required" => "true" }
+  s.metadata = {
+    "bug_tracker_uri" => "https://github.com/activeadmin/activeadmin/issues",
+    "changelog_uri" => "https://github.com/activeadmin/activeadmin/blob/master/CHANGELOG.md",
+    "documentation_uri" => "https://activeadmin.info/documentation.html",
+    "homepage_uri" => "https://activeadmin.info",
+    "mailing_list_uri" => "https://groups.google.com/group/activeadmin",
+    "rubygems_mfa_required" => "true",
+    "source_code_uri" => "https://github.com/activeadmin/activeadmin",
+    "wiki_uri" => "https://github.com/activeadmin/activeadmin/wiki"
```

Comment:
I'd rather only declare what RubyGems doesn't pick up since the rest it's already figuring out correctly, as you alluded to. Thanks.

```suggestion
    "changelog_uri" => "https://github.com/activeadmin/activeadmin/blob/master/CHANGELOG.md",
    "rubygems_mfa_required" => "true"
```

Reply:
Right on ✌️

Reply:
Hi,

I would suggest to reconsider and revert to the full definition

I do not know how rubygems figures out links, but it appears that links are outdated and non-secure

![image](https://github.com/activeadmin/activeadmin/assets/556268/10382553-7cb2-4264-8c92-397b9a8ea099)

I guess that defining them in the gemspec will fix with the secure versions

Reply:
Thanks. That's a good point @tagliala. I'm ok with the original submission then but with the caveat that the `documentation_uri` point to the base domain instead, same as homepage. @Aesthetikx sorry about that. Would you mind redoing the changes?

Reply:
Right on 2.0 ✌️

Reply:
@Aesthetikx thanks!